events/music/error.js

```diff
@@ -6,6 +6,6 @@ module.exports = {
 	once: false,
 	execute(err) {
 		logger.error({ message: 'An error occurred. Quaver will now shut down to prevent any further issues.', label: 'Lavalink' });
-		shuttingDown('', err);
+		shuttingDown('Lavalink Error', err);
```

Comment:
i think it should be `lavalink` instead, don't really need to put any more than that